site/content/en/docs/Examples/_index.md

```diff
@@ -45,3 +45,7 @@ These are all examples of simple game server implementations, that integrate the
 
 - {{< ghlink href="examples/allocation-endpoint" >}}Allocation Endpoint Example{{< /ghlink >}} An Agones clusters aware proxy, built on Google Cloud.
 
+## Integration with Open Match
```

Comment:
As long as you are editing this file, could you add a `-` on line 46 so that its formatting is consistent with the rest of this file?